docs/hub/model-card-annotated.md

```diff
@@ -0,0 +1,304 @@
+# Annotated Model Card Template
+
+
+## Template
+
+[https://github.com/huggingface/huggingface_hub/blob/main/src/huggingface_hub/templates/modelcard_template.md](https://github.com/huggingface/huggingface_hub/blob/main/src/huggingface_hub/templates/modelcard_template.md)
+
+
+## Directions
+
+Fully filling out a model card requires input from a few different roles. (One person may have more than one role.)  We’ll refer to these roles as the **developer**, who writes the code and runs training; the **sociotechnic**, who is skilled at analyzing the interaction of technology and society long-term (this includes lawyers, ethicists, sociologists, or rights advocates); and the **project organizer**, who understands the overall scope and reach of the model, can roughly fill out each part of the card, and who serves as a contact person for model card updates.
+
+* The **developer** is necessary for filling out [Training Procedure](#training-procedure) and [Technical Specifications](#technical-specifications). They are also particularly useful for the “Limitations” section of [Bias, Risks, and Limitations](#bias-risks-andlimitations). They are responsible for providing [Results](#results) for the Evaluation, and ideally work with the other roles to define the rest of the Evaluation: [Testing Data, Factors & Metrics](#testing-data-factors-and-metrics).
+
+* The **sociotechnic** is necessary for filling out “Bias” and “Risks” within [Bias, Risks, and Limitations](#bias-risks-and-limitations), and particularly useful for “Out of Scope Use” within [Uses](#uses).
+
+* The **project organizer** is necessary for filling out [Model Details](#model-details) and [Uses](#uses). They might also fill out
+[Training Data](#training-data). Project organizers could also be in charge of [Citation](#citation), [Glossary](#glossary), 
+[Model Card Contact](#model-card-contact), [Model Card Authors](#model-card-authors), and [More Information](#more-information).
+
+_Instructions are provided below, in italics._
+
+# Model Name
+
+**Section Overview:**  Provide the model name and a 1-2 sentence summary of what the model is. 
+
+{{ model_id | default("Model ID", true) }}
+{{ model_summary | default("", true) }}
+
+# Table of Contents
+**Section Overview:** Provide this with links to each section, to enable people to easily jump around/use the file in other locations with the preserved TOC/print out the content/etc.
+
+# Model Details
+
+**Section Overview:** This section provides basic information about what the model is, its current status, and where it came from. It should be useful for anyone who wants to reference the model.
+
+
+## Model Description
+
+
+{{ model_description | default("", true) }}
```

Comment:
we should remove the curly braces syntax and just leave the textual description for each section, no?

Reply:
I think it's helpful to have them to help people just visually align between what they see in the usual template and what the annotations are. It also tells people the variable name of what the instructions are referring to, which seems useful.